frontend/src/components/RepositoryForm.vue

```diff
@@ -73,7 +73,7 @@
       />
     </div>
     <!-- Project schemaChangeType selector -->
-    <div v-if="isDev">
```

Comment:
We can keep this as it is till the final launch.

Reply:
Reverted.